mimic/test/test_maas.py

```diff
@@ -1797,6 +1797,40 @@ def test_latest_alarm_states(self):
         self.assertEquals(resp.code, 200)
         self.assertEquals(data['values'][0]['latest_alarm_states'][0]['state'], 'CRITICAL')
 
+    def test_change_log(self):
```

Comment:
Please add a docstring, similar to the others in this module - i.e. following https://jml.io/test-docstrings/ - for this test.